addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/EEP.java

```diff
@@ -209,9 +215,9 @@ protected int getDataLength() {
         return 0;
     }
 
-    protected void convertFromCommandImpl(String channelId, String channelTypeId, Command command, State currentState,
-            Configuration config) {
-
+    protected void convertFromCommandImpl(String channelId, String channelTypeId, Command command,
+            Hashtable<String, State> currentState, Configuration config) {
```

Comment:
Does it need to be a `Hashtable` type or is the generic `Map` type enough? The most generic type should be part of a function signature.

Reply:
Thx, changed that!